release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -121,6 +121,44 @@ io:println(nameText); // "DanBrown"
 
 ##### Language Server
 
+##### Ballerina Shell
+
+Ballerina Shell now supports redefining module-level declarations and variable declarations. `/remove NAMES` command can be used to remove one or multiple declarations from snippet memory.
+
+```ballerina
+=$ int i = 3;
+=$ string j = "Hi";
+=$ string i = "Hello";  // Same variable can be redefined
+=$ /remove i j   // Variables can be removed
+=$ i
+| error: undefined symbol 'i'
+|       i
+|       ^
+| Compilation aborted due to errors.
+```
+
+A command-line argument was added with `-f` or `--file` argument name to load module-level declarations within a file. `/file <FILENAME` command can also be used for this purpose, from within the shell. Note that `--force-debug` will now have only a long option; `-f` short option is now used to open the file.
```

Comment:
Shall we emphasize the feature first?

```suggestion
- Ballerina Shell can now load definitions and declarations from a file. The file to load from can be specified using the `-f` or `--file` command-line options when launching Ballerina Shell. Alternatively the `/file` command can also be used for this purpose, from within the shell. 

Note that the `--force-dumb` command-line option will now have only a long option and the short option `-f` is now used to load from a file.
```

Should be `--force-dumb` right? Not `--force-debug`?